google/cloud/forseti/notifier/notifiers/cscc_notifier.py

```diff
@@ -134,7 +139,9 @@ def _transform_for_api(self, violations):
                     'violation_data': violation.get('violation_data')
                 },
                 'source_id': 'FORSETI',
-                'category': 'UNKNOWN_RISK'
+                'category': violation.get('rule_name'),
+                'url': 'table:{}/id:{}'.format(
+                    VIOLATIONS_TABLE, violation.get('id'))
```

Comment:
Is the 'url' property required? I think we can include the id inside 'properties:' above, having a url like this seems weird.

Reply:
Moving this into the `properties` section so this is not clickable.